ThunderCloud.xcodeproj/project.pbxproj

```diff
@@ -1645,7 +1755,7 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				LIBRARY_SEARCH_PATHS = "$(inherited)";
-				MARKETING_VERSION = 1.7.7;
+				MARKETING_VERSION = 2.0.0;
```

Comment:
@BenCShutt yep, it's in this hidden file ☺️

Reply:
Oh yes :P

Reply:
(In answer to your question about bumping version)